### DISCUSSION TITLE
메모 저장은 어디에 되나요

### DISCUSSION THREAD

**Discussion by dorododo** [green1052/DCRefresher-Reborn#3] [Q&A] [answered]
업데이트떄마다 메모 날라가는데 유지하면서 업데이트할 방법은 없나요

**Comment by green1052** [comment, marked as answer]
브라우저 로컬 또는 동기화 저장소에 저장됩니다. [memo.ts#L58](https://github.com/green1052/DCRefresher-Reborn/blob/62441f31f8af4f64b6aba0e67b6c7d2db08031dd/src/core/memo.ts#L58)

메모 기능을 수정 중 데이터 저장 위치를 바꾸었는데
실수로 기존 데이터를 마이그레이션하게 안했습니다.

다음부터는 데이터 유실 없이 업데이트됩니다.
